assignment-1-sql/rui-qu.sql

```diff
@@ -0,0 +1,121 @@
+-- Assignment 1 SQL
+-- https://github.com/carrot-u/carrot-u-docs/blob/master/2019-session/assignments/assignment-1-sql.md
+
+------------------------------------------------
+-- Question 1
+-- Find the most recent completed order where the basket size was > $500.
+SELECT *
+FROM fact_order
+WHERE completed_at is NOT NULL
+AND charge_amount > 500
+ORDER BY completed_at DESC
+LIMIT 1
+
+------------------------------------------------
+-- Question 2
+-- What GMV have we generated, how many orders were made per week in the last 8 weeks in Canada? 
+-- Order the results from the most recent week to the oldest.
+SELECT DATE_TRUNC('week', fact_order.completed_at) AS WEEK, 
+       SUM(fact_order.charge_amount) AS GMV, 
+       COUNT(fact_order.id) AS NUM_ORDERS, 
+FROM fact_order FO
+INNER JOIN countries C
+ON FO.country_id = C.id
+WHERE C.name = 'Canada'
+-- AND FO.completed_at BETWEEN DATEADD(day, -56, CURRENT_DATE) AND DATEADD(day, -1, CURRENT_DATE)
+-- AND fact_order.completed_at BETWEEN DATEADD(week, -7, CURRENT_DATE) AND CURRENT_DATE
+GROUP BY 1
```

Comment:
Hey a small thing, I would recommend not using column numbers in these clauses here. IMO it makes the statements harder to read, it would be easier to read if you use column names 😄